models/migrations/v1_22/v290.go

```diff
@@ -13,5 +13,12 @@ type HookTask struct {
 
 func AddPayloadVersionToHookTaskTable(x *xorm.Engine) error {
 	// create missing column
-	return x.Sync(new(HookTask))
+	if _, err := x.SyncWithOptions(xorm.SyncOptions{
+		IgnoreIndices:    true,
+		IgnoreConstrains: true,
+	}, new(HookTask)); err != nil {
+		return err
+	}
+	_, err := x.Exec("UPDATE hook_task SET payload_version = 1 WHERE payload_version IS NULL")
```

Comment:
I think we need to switch these statements around.
Otherwise, it seems inconsistent.
Why is it even necessary, even though it wasn't previously?

Reply:
This maybe an unrelated change to this PR. I just found `v290` missed the update operations. Because creating a new column with default value will not update the old record automatically. It should be a bug of previous PR.

Reply:
Maybe split this change out into separate PR?

Reply:
I think fixing it here is fine ... just add a doctor command to address this things ?